dotCMS/src/main/java/com/dotcms/rest/WebResource.java

```diff
@@ -861,7 +870,12 @@ public InitDataObject init() {
                 Logger.debug(InitBuilder.class,
                         () -> request.getRequestURI()+" calling webResource.init(..) with AnonymousAccess set to `" + anonAccess + "`");
             }
-            
+
+            if(requireLicense){
+                if (LicenseUtil.getLevel() < LicenseLevel.STANDARD.level) {
```

Comment:
Codacy found an issue: [These nested if statements could be combined](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5883793)